apps/www/registry/default/ui/spinner.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from "react"
+
+import { cn } from "@/lib/utils"
+
+interface SpinnerBar {
+  animationDelay: number
+  rotate: number
+}
+
+const Spinner = React.forwardRef<
+  HTMLDivElement,
+  React.HTMLAttributes<HTMLDivElement>
+>(({ className, ...props }, ref) => {
+  const spinnerBars: SpinnerBar[] = [
+    { animationDelay: -1.2, rotate: 0.0001 },
+    { animationDelay: -1.1, rotate: 30 },
+    { animationDelay: -1, rotate: 60 },
+    { animationDelay: -0.9, rotate: 90 },
+    { animationDelay: -0.8, rotate: 120 },
+    { animationDelay: -0.7, rotate: 150 },
+    { animationDelay: -0.6, rotate: 180 },
+    { animationDelay: -0.5, rotate: 210 },
+    { animationDelay: -0.4, rotate: 240 },
+    { animationDelay: -0.3, rotate: 270 },
+    { animationDelay: -0.2, rotate: 300 },
+    { animationDelay: -0.1, rotate: 330 },
+  ]
```

Comment:
```suggestion
const spinnerBars: SpinnerBar[] = Array.from({ length: 12 }, (_, index) => ({
  animationDelay: -1.2 + 0.1 * index,
  rotate: 30 * index
}));
```

Little cleaner implementations of the `spinnerBars`?  Could also move it outside the context of the Component to save some performance 👍